src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -205,6 +205,52 @@ export const PsvTechRecord: FormNode = {
 
       type: FormNodeTypes.CONTROL,
       viewType: FormNodeViewTypes.STRING
+    },
+    {
+      name: 'approvalSection',
+      label: 'Type approval',
+      type: FormNodeTypes.SECTION
+    },
+    {
+      name: 'approvalType',
+      label: 'Approval type',
+      type: FormNodeTypes.CONTROL
+    },
+    {
+      name: 'approvalTypeNumber',
+      label: 'Approval type tumber',
+      type: FormNodeTypes.CONTROL
+    },
+    {
+      name: 'ntaNumber',
+      label: 'National type number',
+      type: FormNodeTypes.CONTROL
+    },
+    {
+      name: 'coifSerialNumber',
+      label: 'COIF Serial number',
+      type: FormNodeTypes.CONTROL
+    },
+    {
+      name: 'coifCertifierName',
+      label: 'COIF Certifier name',
+      type: FormNodeTypes.CONTROL
+    },
+    {
+      name: 'coifDate',
+      label: 'COIF Certifier date',
+      type: FormNodeTypes.CONTROL,
+      viewType: FormNodeViewTypes.DATE
+    },
+    {
+      name: 'variantNumber',
+      label: 'Variant number',
+      type: FormNodeTypes.CONTROL
+    },
+    {
+      name: 'variantVersionNumber',
+      label: 'Variant version number',
+      type: FormNodeTypes.CONTROL
```

Comment:
Isn't this the tyre approval section? I think on develop this already exists as it's own template?

Reply:
Don't know why this came through in the rebase. I didn't do this work.

Reply:
Yeah odd it's come through, I think it was like this by mistake and then we removed it in a PR afterwards, so maybe you rebased before that change and just need to do it again?

Reply:
Re-rebased and it's still here :(

Reply:
Code deleted @naathanbrown